R/tostring.R

```diff
@@ -36,29 +36,54 @@ default_hsep <- d_hsep_factory()
 
 
 #' @rdname tostring
+#'
 #' @inheritParams MatrixPrintForm
-#' @param widths (proposed) widths for the columns of \code{x}
+#' @param widths numeric (or NULL). (proposed) widths for the columns of
+#'  \code{x}. The expected length of this numeric vector can be retrieved with
+#'  `ncol() + 1` as the column of row names must also be considered.
 #' @param hsep character(1). Characters to repeat to create header/body
-#' separator line.
-#' @exportMethod toString
+#'  separator line.
+#' @param tf_wrap logical(1). Should the texts like title and footnotes be wrapped?
+#' @param max_width integer(1) (or NULL). If NULL, it will be assigned to
+#'  `options("width")`. Otherwise, it is also possible to use `c("auto")` to
+#'  automatically use the table width.
+#'
+#' @details When wrapping texts such as title, subtitle, or footnotes, please
```

Comment:
This text will need to be massaged but I can do that after merge.

Reply:
Yes of course it has to be completed into the bigger picture of the function